src/beacons/BeaconsContext.tsx

```diff
@@ -180,7 +180,7 @@ const BeaconsContextProvider: React.FC = ({children}) => {
       }
 
       const permissions = await allowedPermissionsForBeacons();
-      if (consentGranted && permissions && !beaconsInfo?.isStarted) {
+      if (consentGranted && permissions.length > 0 && !beaconsInfo?.isStarted) {
```

Comment:
Permissions is an array and returned as empty if no permissions were granted, the condition only checks if there is an array. Then it sends empty to the SDK when starts and fails internally.